Misc/NEWS.d/next/Library/2023-05-01-18-53-20.gh-issue-102140._4gFLu.rst

```diff
@@ -0,0 +1,3 @@
+Enhanced the :meth:`has_header` in the :mod:`CSV` for better header detection.
```

Comment:
- Describe actions like commands, not past things: `Enhance / improve / change X to do Y`, not `enhanced`
- This reads «The `has_header` in the `csv` → add some nouns 😉 
- Lower-case `csv` for the module, full name for the method or the link won’t work